client-app/src/examples/portfolio/detail/OrdersPanelModel.js

```diff
@@ -39,6 +41,28 @@ export class OrdersPanelModel extends HoistModel {
             persistWith: {...PERSIST_DETAIL, path: 'ordersGrid'},
             columns: [
                 {...symbolCol, pinned: true},
+                {
```

Comment:
nit -- might put this in the column/positions file under the name `tradingVolSparklineCol` for clarity